examples/manifold/plot_tsne_faces.py

```diff
@@ -0,0 +1,148 @@
+"""
+===================================
+Comparison of t-SNE methods and options
+===================================
+
+A comparison of t-SNE applied to the Olivetti faces data. The 'exact' and
+'barnes-hut' methods are visualized, as well as init='pca' and init='random'.
+In addition to varying method & init, the script also compares preprocessing
+the input data by reducing the dimensionality via PCA and also varying the
+perplexity.
+
```

Comment:
Great! Will do.

I think it's fine to say the initializing with PCA preserved more of the global structure -- LvdM mentions this in his papers.